docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -268,14 +275,19 @@ public class CustomTenantResolver implements TenantResolver {
 }
 ----
 
-You can define multiple tenants in your configuration file, just make sure they have a unique alias so that you can map them properly when resolving a tenant from your `TenantResolver` implementation.
+You can define multiple tenants in your configuration file.
+To map them correctly when resolving a tenant from your `TenantResolver` implementation, ensure each has a unique alias.
 
-However, using a static tenant resolution (configuring tenants in `application.properties` and resolving them with `TenantResolver`) prevents testing the endpoint with `Dev Services for Keycloak` since `Dev Services for Keycloak` has no knowledge of how the requests will be mapped to individual tenants and can not dynamically provide tenant-specific `quarkus.oidc.<tenant-id>.auth-server-url` values and therefore using `%prod` prefixes with the tenant-specific URLs in `application.properties` will not work in test or dev mode.
+However, using a static tenant resolution method, such as configuring tenants in `application.properties` and resolving them with `TenantResolver`, presents limitations.
+This approach is not suitable for testing the endpoint with Dev Services for Keycloak.
+The reason is that Dev Services for Keycloak does not understand how requests are mapped to individual tenants.
+Therefore, it cannot dynamically provide tenant-specific `quarkus.oidc.<tenant-id>.auth-server-url` values.
+As a result, using `%prod` prefixes with the tenant-specific URLs in `application.properties` is ineffective in tests or development mode.
 
 [NOTE]
 ====
-When a current tenant represents an OIDC `web-app` application, the current `io.vertx.ext.web.RoutingContext` will contain a `tenant-id` attribute by the time the custom tenant resolver has been called for all the requests completing the code authentication flow and the already authenticated requests, when either a tenant specific state or session cookie already exists.
-Therefore, when working with multiple OpenID Connect Providers, you only need a path specific check to resolve a tenant id if the `RoutingContext` does not have the `tenant-id` attribute set, for example:
+When a current tenant represents an OIDC `web-app` application, the current `io.vertx.ext.web.RoutingContext` contains a `tenant-id` attribute by the time the custom tenant resolver has been called for all the requests completing the code authentication flow and the already authenticated requests, when either a tenant-specific state or session cookie already exists.
```

Comment:
@pedroigor, can I get your help rewriting this line so that it is easier to understand?
I can't make sense of it.
Can we break it into multiple statements?

Reply:
@sberyozkin I think we can enhance this line in a future PR. Closing this for now.